app/src/src-SessionBear/components/InstallExtensionCTA/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { Row, Space, Typography } from "antd";
+import { Alert } from "antd";
+import UAParser from "ua-parser-js";
+import { actions } from "store";
+import { supportedBrowserExtensions } from "./constants/supportedBrowserExtensions";
+import APP_CONSTANTS from "../../../config/constants";
+import { InstallExtensionContent } from "./type";
+import { trackExtensionInstallationButtonClicked } from "modules/analytics/events/common/onboarding/index";
+import "./installExtensionCTA.css";
+
+const HEADING = "Install Browser extension to record sessions for faster debugging and bug reporting";
+const SUBHEADING =
+  "Safely capture mouse movement, console, network & environment data automatically on your device for sharing and debugging. Private and secure, works locally on your browser.";
+
+const InstallExtensionCTA: React.FC<InstallExtensionContent> = ({
+  eventPage,
+  supportedBrowsers = null,
+  isUpdateRequired = false,
+  heading = HEADING,
+  subHeading = SUBHEADING,
+}) => {
+  const dispatch = useDispatch();
+  const [isReloadPagePromptVisible, setIsReloadPagePromptVisible] = useState(false);
+  const [browser, setBrowser] = useState<
+    { downloadURL: string; name: string; iconURL: string; alt: string } | undefined
+  >();
+
+  useEffect(() => {
+    const parser = new UAParser();
+    parser.setUA(window.navigator.userAgent);
+    const result = parser.getResult();
+    const browserDetected = result.browser.name;
+    let currentBrowserData;
+
+    supportedBrowserExtensions.forEach((extensionData) => {
+      // @ts-ignore
+      if (!supportedBrowsers || supportedBrowsers.includes(extensionData.name)) {
+        if (extensionData.name === browserDetected) {
+          currentBrowserData = extensionData;
+        }
+      }
+    });
+
+    setBrowser(currentBrowserData);
+  }, [supportedBrowsers]);
+
+  const handleDownloadExtensionClick = () => {
+    dispatch(actions.updateExtensionInstallSource(window.location.pathname));
+    setIsReloadPagePromptVisible(true);
+    trackExtensionInstallationButtonClicked(eventPage);
```

Comment:
just an example of why I mentioned [this](https://github.com/requestly/requestly/pull/1864#discussion_r1654655318)

How will we know which extension install click event this is